### PR TITLE
tools: add description for task auto restore after dm-worker restart

### PR DESCRIPTION
**Pull request by amyangfei**
refer to DM update: https://github.com/pingcap/dm/pull/88
Besides we also need to update this in English version docs

### CODE REVIEW

**Comment by amyangfei** [comment]
PTAL @GregoryIan @lilin90

File: tools/dm/cluster-operations.md [hunk 1]
```diff
@@ -39,21 +39,21 @@ $ ansible-playbook stop.yml
...
 
-    如果 DM-worker 上运行的任务未启用 sharding DDL 同步功能，DM-worker 重启时会检查断点信息。您可以使用 [`start-task` 命令](/tools/dm/practice.md#第-4-步启动任务)自动恢复数据同步。
+    如果 DM-worker 上运行的任务未启用 sharding DDL 同步功能，DM-worker 重启时会检查断点信息和本地记录的任务状态，重启前处于运行中状态的任务会自动恢复数据同步。
 
 + 已启用 sharding DDL 同步
 
     - DM 同步 sharding DDL 语句时，如果 DM-worker 成功执行（或跳过）sharding DDL 的 binlog event，与 DM-worker 中的 sharding DDL 语句相关的所有表的断点信息都会被更新至 DDL 语句对应的 binlog event 之后的位置。
 
-    - 当 DM-worker 重启发在 sharding DDL 语句同步开始前或完成后，您可以使用 `start-task` 命令，根据断点信息自动恢复数据同步。
+    - 当 DM-worker 重启发在 sharding DDL 语句同步开始前或完成后，DM-worker 会根据断点信息和本地记录的任务状态自动恢复数据同步。
```
Comment (yikeke) [inline tools/dm/cluster-operations.md]:
```suggestion
    - 当 DM-worker 重启发生在 sharding DDL 语句同步开始前或完成后，DM-worker 会根据断点信息和本地记录的任务状态自动恢复数据同步。
```

File: tools/dm/cluster-operations.md [hunk 2]
```diff
@@ -39,21 +39,21 @@ $ ansible-playbook stop.yml
 
 **全量数据导入过程中：**
 
-对于全量数据导入时的 SQL 文件，DM 使用下游数据库记录断点信息。DM-worker 重启时会检查断点信息。您可以使用 [`start-task` 命令](/tools/dm/practice.md#第-4-步启动任务)自动恢复数据同步。
+对于全量数据导入时的 SQL 文件，DM 使用下游数据库记录断点信息，DM-worker 会在本地文件记录任务状态。DM-worker 重启时会检查断点信息和本地记录的任务状态，重启前处于运行中状态的任务会自动恢复数据同步。
```
Comment (GregoryIan) [inline tools/dm/cluster-operations.md]:
```suggestion
对于全量数据导入时的 SQL 文件，DM 使用下游数据库记录断点信息，DM-worker 会在本地 meta 文件记录自任务配置信息。DM-worker 重启时会检查断点信息和本地记录的自任务 meta 信息，重启前处于运行中状态的任务会自动恢复数据同步。
```

File: tools/dm/cluster-operations.md [hunk 3]
```diff
@@ -39,21 +39,21 @@ $ ansible-playbook stop.yml
...
 
 **增量数据同步过程中：**
 
 对于增量数据导入过程中的 binlog，DM 使用下游数据库记录断点信息，并会在同步任务开始或恢复后的第一个五分钟之内开启安全模式。
 
 + 未启用 sharding DDL 同步
 
-    如果 DM-worker 上运行的任务未启用 sharding DDL 同步功能，DM-worker 重启时会检查断点信息。您可以使用 [`start-task` 命令](/tools/dm/practice.md#第-4-步启动任务)自动恢复数据同步。
+    如果 DM-worker 上运行的任务未启用 sharding DDL 同步功能，DM-worker 重启时会检查断点信息和本地记录的任务状态，重启前处于运行中状态的任务会自动恢复数据同步。
```
Comment (GregoryIan) [inline tools/dm/cluster-operations.md]:
```suggestion
    如果 DM-worker 上运行的任务未启用 sharding DDL 同步功能，DM-worker 重启时会检查断点信息和本地记录的子任务信息，重启前处于运行中状态的任务会自动恢复数据同步。
```

File: tools/dm/cluster-operations.md [hunk 4]
```diff
@@ -39,21 +39,21 @@ $ ansible-playbook stop.yml
...
 
 + 已启用 sharding DDL 同步
 
     - DM 同步 sharding DDL 语句时，如果 DM-worker 成功执行（或跳过）sharding DDL 的 binlog event，与 DM-worker 中的 sharding DDL 语句相关的所有表的断点信息都会被更新至 DDL 语句对应的 binlog event 之后的位置。
 
-    - 当 DM-worker 重启发在 sharding DDL 语句同步开始前或完成后，您可以使用 `start-task` 命令，根据断点信息自动恢复数据同步。
+    - 当 DM-worker 重启发生在 sharding DDL 语句同步开始前或完成后，DM-worker 会根据断点信息和本地记录的任务状态自动恢复数据同步。
```
Comment (GregoryIan) [inline tools/dm/cluster-operations.md]:
```suggestion
    - 当 DM-worker 重启发生在 sharding DDL 语句同步开始前或完成后，DM-worker 会根据断点信息和本地记录的子任务信息自动恢复数据同步。
```

**Pull request merged**